src/Api/HL/Middleware/IPRestrictionRequestMiddleware.php

```diff
@@ -81,12 +77,12 @@ public function process(MiddlewareInput $input, callable $next): void
         $next($input);
     }
 
-    private function isIPAllowed(string $ip, string $allowed_ips): bool
+    public function isIPAllowed(string $ip, string $allowed_ips): bool
```

Comment:
You can keep this method private and use the `$this->callPrivateMethod()` helper in tests.